aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/migrations/AddModelNameToModelMetadataKey.java

```diff
@@ -30,6 +31,7 @@
 /**
  * Add model name to ModelMetadata key.
  */
+@InternalApiWarning
```

Comment:
I'd prefer that we don't go adding this annotation to existing public classes. It can give the impression later on that this annotation has always been there and that we are free to break the contract, which isn't the case.